src/web/routes/application/common/formatters/errors.js

```diff
@@ -0,0 +1,19 @@
+const mergeErrorMessage = (message, cause) => cause ? `${message}. ${cause.toString()}` : message
+
+const mergeErrorStack = (error, cause) => cause ? `${error.stack}\nCaused by: ${cause.stack}` : error.stack
+
+const mergeErrorProps = ({ cause, message } = {}) => {
+  const error = new Error()
+  Error.captureStackTrace(error, mergeErrorProps)
```

Comment:
This sets the root of the stack trace as the point at which `mergeErrorProps` was called, e.g. `post.js`